ideal-voting-server/src/main/scala/cz/idealiste/idealvoting/server/Voting.scala

```diff
@@ -35,20 +35,21 @@ class Voting(config: Config.Voting, db: Db, random: Random.Service, clock: Clock
     ElectionViewAdmin(election.metadata, election.admin, election.options, voters)
   }
 
-  def createElection(create: CreateElection): Task[(String, String)] = {
+  def createElection(create: CreateElection, now: OffsetDateTime): Task[(String, String)] = {
     for {
       adminToken <- generateToken
       voters <- create.voters.traverse { email =>
         generateToken.map(Voter(email, _, voted = false))
       }
-      now <- clock.currentDateTime
       titleMangled = mangleTitle(create.title)
       electionMetadata = ElectionMetadata(create.title, titleMangled, create.description, now, None)
       admin = Admin(create.admin, adminToken)
       options = create.options.zipWithIndex.map { case (CreateOption(title, description), id) =>
         BallotOption(id, title, description)
       }
-      _ <- db.createElection(electionMetadata, admin, options, voters)
+      () <- logger.debug(s"Creating election $electionMetadata with admin $admin, voters $voters and options $options.")
```

Comment:
@radim91 The secret tokens will be logged, so you can read the log and see what they are and then can use them for testing.
Here's where the logging takes place.